EIPS/eip-6110.md

```diff
@@ -76,11 +76,51 @@ Beginning with the `FORK_BLOCK`, each deposit accumulated in the block **MUST**
 in the order they appear in the logs. To illustrate:
 
 ```python
-def parse_deposit_data(deposit_event_data) -> bytes[]:
-  """
-  Parses deposit data from DepositContract.DepositEvent data
-  """
-  pass
+def parse_deposit_data(deposit_event_data):
+    """
+    Parses deposit data from DepositContract.DepositEvent data.
+    """
+
+    # TODO: verify byte length of deposit_event_data? Should be 576. Should this be part of the spec? (Yes?)
+    
+    """ 
+        The deposit event is encoded as:
+            event DepositEvent(
+                bytes pubkey,
+                bytes withdrawal_credentials,
+                bytes amount,
+                bytes signature,
+                bytes index
+            );
+    """
+
+    # Start offset of data reading skips over the first 5 32-byte values
+    # which represents the dynamic size of the encoded bytes event data
+    # The size itself is encoded in a 32-byte, so skip over this as well 
+    # to arrive at the first actual data offset to be read (pubkey)
+    currentOffset = 32 * 5 + 32
```

Comment:
Can we have PUBKEY_OFFSET, WITHDRAWAL_CREDENTIALS_OFFSET, etc instead of these computations? and then:
```python
pubkey = deposit_event_data[PUBKEY_OFFSET:PUBKEY_OFFSET + 48]
withdrawal_credentials = deposit_event_data[WITHDRAWAL_CREDENTIALS_OFFSET: WITHDRAWAL_CREDENTIALS_OFFSET + 32]
...
```

Reply:
Have added! Please verify if my offsets and sizes are correct :smile: :+1: